test/Prototypes/CollectionTransformers.swift

```diff
@@ -12,6 +12,9 @@
 // RUN: %target-run-stdlib-swift
 // REQUIRES: executable_test
 
+// FIXME: swift-3-indexing-model: This prototype needs to be updated to new indexing model
```

Comment:
I thought this was a code prototype for some not yet implemented feature so with that in mind I decide to knock out the code to suppress compile issues while leaving it to fail so we wouldn't forget about it, etc.

I can uncomment.